package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "base",
   "private": true,
-  "version": "8.8.4",
+  "version": "8.8.5",
```

Comment:
Would put this to 8.9.0 rather than patch version bump, just due to the overall layout changes that happened